doc/src/manual/strings.md

```diff
@@ -254,6 +255,9 @@ julia> substr = SubString(str, 1, 4)
 
 julia> typeof(substr)
 SubString{String}
+
+julia> @views typeof(str[1:4]) # @views converts slices to SubStrings
```

Comment:
Might be a bit clearer to put the `@views` inside the `typeof`?

Reply:
The **whole point** of `@views` is that you *don't* need to put it directly adjacent to the indexing expression (unlike `@view`).

I think many people aren't aware of this feature of `@views`, and so it is important to use it differently from `@view` in examples.